client/src/components/Explore.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { Grid } from "@material-ui/core";
+import { useTheme, makeStyles } from "@material-ui/core/styles";
+import ExploreCard from "./ExploreCard";
+
+const exploreStyles = makeStyles((theme) => ({
+	pageContainer: {
+		overflow: "hidden",
+		marginBottom: "3rem",
+	},
+	pt4: {
+		paddingTop: "3rem",
+	},
+
+	title: {
+		fontSize: "2.5rem",
+	},
+	subTitle: {
+		color: `${theme.palette.primary.light}`,
+		marginTop: "-0.25rem",
+		fontSize: "19px",
+	},
+	cardContainer: {
+		paddingTop: "3rem",
+		paddingLeft: "10%",
+		paddingRight: "10%",
+	},
+}));
+
+function Explore() {
+	//To get image for exploreCard, make sure the image is saved in public with the name of the place. ex: "public/images/cancun.png"
+	//THIS: props.location.toLowerCase() === saved image name
+	const theme = useTheme();
+	const classes = exploreStyles(theme);
+
+	return (
+		<Grid
+			container
+			justify="center"
+			alignItems="center"
+			direction="row"
+			className={classes.pageContainer}
+		>
+			<Grid
+				container
+				item
+				xs={12}
+				justify="center"
+				alignItems="center"
+				className={classes.pt4}
+			>
+				<Grid
+					item
+					xs={12}
+					container
+					justify="center"
+					alignItems="center"
+				>
+					<h1 className={classes.title}>Explore destinations</h1>
```

Comment:
Material UI has a typography component that inherits styling from the theme.  May be better to use than these specific elements for consistency.  For example the `World's Top Destinations to Explore` should probably be an h2 or h3, as opposed to a paragraph element.

Reply:
Sounds good, thanks for the comment 💯  
I'll look through the components I made and convert